dartagnan/src/main/java/com/dat3m/dartagnan/encoding/PropertyEncoder.java

```diff
@@ -350,9 +423,8 @@ public BooleanFormula encodeDeadlocks() {
                 allStuckOrDone = bmgr.and(allStuckOrDone, bmgr.or(isStuck, isTerminatingNormally));
             }
 
-            final BooleanFormula deadlockVariable = LIVENESS.getSMTVariable(context);
             final BooleanFormula hasLivenessViolation = bmgr.and(allStuckOrDone, atLeastOneStuck);
```

Comment:
Maybe `deadlockViolation` makes more sense now

Reply:
You mean `hasDeadlock` maybe? `deadlockViolation` sounds kinda wrong.